httpe/httpe.go

```diff
@@ -0,0 +1,51 @@
+// Package httpe holds http server utilities
+//nolint: gochecknoglobals
```

Comment:
I think this will become part of the godocs for the package, so i might be best to move either before the doc (with a blank line) or after the `package` declaration.

Reply:
done. (added to .golangci.yaml)